client/src/components/Visualiser/Visualiser.tsx

```diff
@@ -17,14 +17,13 @@ interface Props {
  * Given the `topicTitle`, this component will render the corresponding
  * visualiser and load up the commands for that visualiser.
  */
-const Visualiser: React.FC<Props> = ({ topicTitle }) =>
-  topicTitle ? (
-    <Pane orientation="horizontal" minSize={150.9}>
-      <VisualiserCanvas topicTitle={topicTitle} />
-      <VisualiserManager topicTitle={topicTitle} />
-    </Pane>
-  ) : (
-    <CircularLoader />
-  );
+const Visualiser: React.FC<Props> = ({ topicTitle }) => (topicTitle ? (
+  <Pane orientation="horizontal" minSize={150.9}>
```

Comment:
Why 150.9? Should we turn this into a constant?

Reply:
No clue why, i didnt write the code xd. I think if we put all the visualiser stuff in visualiser-src then we could have a settings file for the visualiser component and then keep the settings files for each visualiser of course

Reply:
I wonder who wrote this garbage.

The minSize I believe just sets the smallest size you can set a pane to be (without collapsing it). Having it as a constant is a good idea and setting it to a not weird number.